src/renderer/views/components/equalizer.ejs

```diff
@@ -237,15 +237,15 @@
                         "import": {
                             "icon": "./assets/feather/share.svg",
                             "name": app.getLz('action.import'),
-                            "action": function () {
-                                this.importPreset()
```

Comment:
Nani?

Reply:
Is that implemented?
I thought it wasn't because it did nothing when I clicked on it.

Reply:
`[21560:0303/204625.071:INFO:CONSOLE(1906)] "TypeError: this.importPreset is not a function", source: http://localhost:9000/lib/vue.dev.js (1906)`

I fixed this end up the whole app was hanged when I clicked on it.
`[20:54:08.097] [warn]  WebContents #1 called ipcRenderer.sendSync() with 'import-eq' channel without listeners.`